examples/expressions/10_expressions_intro.py

```diff
@@ -251,6 +258,25 @@
 # %%
 print(predictions.skb.describe_param_grid())
 
+# %%
+# We can first run a small dry-run to check if our param search will run correctly:
+
+# %%
+quick_search = predictions.skb.get_randomized_search(
+    scoring="roc_auc",
+    n_iter=4,
+    n_jobs=4,
+    random_state=0,
+    fitted=True,
+    subsampling=True,
```

Comment:
Maybe add a comment like this to draw attention to the parameter? 
```suggestion
    subsampling=True, # force the randomized search to run only on the subsample
```

Reply:
thanks!